openc3-cosmos-cmd-tlm-api/app/controllers/model_controller.rb

```diff
@@ -26,6 +26,9 @@ class ModelController < ApplicationController
   def index
     return unless authorization('system')
     render json: @model_class.names(scope: params[:scope])
+  rescue StandardError => error
+    render json: { status: 'error', message: error.message }, status: 500
+    OpenC3::Logger.error(error.formatted)
```

Comment:
Logger errors cause messages in CmdTlmServer - Probably don't want that here.   Change to using rails logger.